_episodes/02-installation.md

```diff
@@ -22,61 +22,57 @@ be found in the
 We will first install Conda, and then ESMValTool. We end this chapter by testing
 that the installation was successful.
 
-## Install Conda
+## Install ESMValTool on Windows
 
-ESMValTool is distributed using [Conda](https://conda.io/). We will be using the
-Miniconda minimal installer for conda. We suggest a Python 3 based installer,
-though if you happen to already have Conda installed it should also work with
-Python 2. For more information about installing conda, see [the conda
-installation
-documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).
+ESMValTool does not directly support Windows, but succesful usage has been
+reported through the [Windows Subsystem for
+Linux(WSL)](https://docs.microsoft.com/en-us/windows/wsl/), available in Windows
+10.
 
-### Linux
+To install the WSL please follow the instructions [on the Windows Documentation
+page](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
 
-1.  Please download Miniconda3 by running:
+After installing the WSL, installation can be done using the same instructions for ``Linux``.
 
-    ```bash
-    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
-    ```
 
-It is also possible to download Miniconda3 for Linux on the [miniconda page](https://docs.conda.io/en/latest/miniconda.html). If you have problems with the 64 bit version in the next step(s) you can alternatively try a 32 bit version.
+## Install ESMValTool on Linux/MacOSX
 
-2.  Next, run the installer from the place where you downloaded it
+### Install Conda
 
-    ```bash
-    bash Miniconda3-latest-Linux-x86_64.sh
-    ```
+ESMValTool is distributed using [Conda](https://conda.io/).
+Let's check if we already have Conda installed by running:
 
-3.  Follow the instructions in the installer. The defaults should normally
-    suffice.
+```bash
+conda list
+```
 
-4.  You will need to restart your terminal for the changes to have effect.
+If conda is installed, we will see a list of packages.
+We recommend updating conda before esmvaltool installation. To do so, run:
 
-5.  Verify you have a working conda installation by listing all installed
-    packages
+```bash
+conda update
```

Comment:
Should this be `conda update -n base conda`?

Reply:
@Peter9192 thanks. It is fixed.